mix.exs

```diff
@@ -5,12 +5,12 @@ defmodule Osabie.MixProject do
     [
       app: :osabie,
       version: "1.0.1",
-      elixir: ">= 1.6.0",
+      elixir: ">= 1.9.0",
```

Comment:
The version in the [workflow.yml](https://github.com/Adriandmen/05AB1E/blob/master/.github/workflows/workflow.yml#L23) needs to be adjusted for the CI to work properly.

The minimum supported version in the **README.md** should also be updated as well.